doc/source/usage.rst

```diff
@@ -26,4 +34,7 @@ With that said, there are a few universal (Pypiper-added) options that are frequ
   	Force run follow-functions. By default, follow-functions are only run if their corresponding ``run`` command was run; with this option you can force all follow functions to run. This is useful for regenerating QC data on existing output. For more details, see :ref:`the follow argument <the_follow_argument>`.
```

Comment:
Does sphinx support quoting the...whatever you call the text for the menu items on the left sidebar? If so, it'd be nice to quote `follow` but no big deal.

Reply:
It would just underscore the fact that it's a feature if interest to explore, when glancing at the side menu.

Reply:
yep, you can do that